controllers/component_build_controller_pac.go

```diff
@@ -72,6 +72,17 @@ const (
 	defaultPipelineBundle = "quay.io/redhat-appstudio-tekton-catalog/pipeline-docker-build:8cf8982d58a841922b687b7166f0cfdc1cc3fc72"
 )
 
+var mergeRequestDescription = strings.ReplaceAll(
+	`# Pipelines as Code configuration proposal
+
+To start the PipelineRun, add a new comment with content ~/ok-to-test~
+
+For more detailed information about running a PipelineRun, please refer to Pipelines as Code documentation [Running the PipelineRun](https://pipelinesascode.com/docs/guide/running/)
+
+To customize the proposed PipelineRuns after merge, please refer to [Build Pipeline customization](https://redhat-appstudio.github.io/docs.stonesoup.io/Documentation/main/getting-started/build_service.html)
+`,
+	"~", "`")
```

Comment:
would it make sense to rather added it to const?
```suggestion
const mergeRequestDescription = `# Pipelines as Code configuration proposal

To start the PipelineRun, add a new comment with content ` + "`/ok-to-test`" + `

For more detailed information about running a PipelineRun, please refer to [Pipelines as Code documentation](https://pipelinesascode.com/docs/guide/running/)

To customize the proposed PipelineRuns after merge, please refer to [Build Pipeline customization](https://redhat-appstudio.github.io/docs.stonesoup.io/Documentation/main/getting-started/build_service.html)`
```

Reply:
Updated